maven-javadoc-plugin/pom.xml

```diff
@@ -148,7 +148,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-archiver</artifactId>
-      <version>2.5</version>
+      <version>2.6</version>
```

Comment:
Do you want a separate jira issue but include this change only in this PR?

Or do you want a new PR/issue with this change only?

Or do you want me to revert this version change ( which will cause a **maven-archiver** directory to be created in the root of the project after the tests run ).